planning/autoware_obstacle_cruise_planner/src/node.cpp

```diff
@@ -387,12 +432,17 @@ ObstacleCruisePlannerNode::ObstacleCruisePlannerNode(const rclcpp::NodeOptions &
   debug_slow_down_planning_info_pub_ =
     create_publisher<Float32MultiArrayStamped>("~/debug/slow_down_planning_info", 1);
 
+  // tf listener
+  tf_buffer_ = std::make_unique<tf2_ros::Buffer>(get_clock());
+  tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*tf_buffer_);
+
   const auto longitudinal_info = LongitudinalInfo(*this);
 
   ego_nearest_param_ = EgoNearestParam(*this);
 
   enable_debug_info_ = declare_parameter<bool>("common.enable_debug_info");
   enable_calculation_time_info_ = declare_parameter<bool>("common.enable_calculation_time_info");
+  use_pointcloud_ = declare_parameter<bool>("common.use_pointcloud");
```

Comment:
Can you remove this variable `use_pointcloud_`...? I guess the logical product of use_pointcloud_for_stop_ and use_pointcloud_for_slow_down_ represents same meaning.

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6907/commits/f2dc8cf5dc8e8521abf7009124b33171a52a9533